front/app/containers/ProjectsShowPage/shared/header/utils.ts

```diff
@@ -22,34 +20,20 @@ export const hasEmbeddedSurvey = (
   return hasSurveyPhase;
 };
 
-export const hasSurveyWithAnyonePermissions = (
-  projectPermissions: IPCPermissions | undefined,
-  phasesPermissions:
-    | UseQueryResult<IPCPermissions, unknown>[]
-    | undefined
-    | null
+export const hasNativeSurvey = (
+  project: IProjectData,
+  phases: TPhases | null
 ) => {
-  let hasSurveyWithAnyonePermissions = false;
-
-  if (!isNilOrError(phasesPermissions) && phasesPermissions.length > 0) {
-    phasesPermissions.map((permissions) => {
-      if (permissions.data && permissions.data.data) {
-        permissions.data.data.map((permission) => {
-          if (permission.attributes.permitted_by === 'everyone') {
-            hasSurveyWithAnyonePermissions = true;
-          }
-        });
+  let hasSurveyPhase = false;
+  if (!isNilOrError(phases)) {
+    phases.forEach((phase) => {
+      if (phase.attributes.participation_method === 'survey') {
```

Comment:
This checks for external surveys actually - we need to use 'native_survey' instead :)

Reply:
Good catch, forgot to change this while copying!